src/shared/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Tooltip, { TooltipProps } from './Tooltip'
+import { Text, Button } from '@/shared/components'
+import { Regular as ChannelPreview } from '../ChannelPreview/ChannelPreview.stories'
+import { Meta, Story } from '@storybook/react'
+
+export default {
+  title: 'Shared/Tooltip',
+  component: Tooltip,
+  argTypes: {
+    text: { control: 'text', defaultValue: 'Lorem ipsum dolor sit amet!' },
+    arrowDisabled: { control: 'boolean', defaultValue: false },
+  },
+} as Meta
+
+const TextTooltip: Story<TooltipProps> = ({ text, arrowDisabled }) => (
+  <Tooltip text={text} arrowDisabled={arrowDisabled}>
```

Comment:
If we use this instead, we don't need to update the story if we add some new props:

```suggestion
const TextTooltip: Story<TooltipProps> = (args) => (
  <Tooltip {...args}>
```